pywr/parameters/parameters.py

```diff
@@ -130,6 +130,27 @@ def load(cls, model, data):
 InterpolatedVolumeParameter.register()
 
 
+class InterpolatedLevelParameter(AbstractInterpolatedParameter):
+    """
+    Generic interpolation parameter for a non-storage node. It interpolates a value based on node flow at previous timestep
+    """
+    def __init__(self, model, node, x, y, interp_kwargs=None, **kwargs):
+        super(InterpolatedLevelParameter, self).__init__(model, x, y, interp_kwargs, **kwargs)
```

Comment:
With Python 3.6+ we can simplify this to: `super().__init__(model, x, y, interp_kwargs, **kwargs)`

Reply:
Done